src/main/resources/static/app.js

```diff
@@ -74,5 +74,12 @@ $(function () {
         })
     }), $("#send").click(function () {
         sendMessage()
+    }), $("#name").on('keyup', function () {
```

Comment:
If the user simply clicks on the `connect to` button without touching the `username` field then it's allowing.

Reply:
Oh I will change it...

Reply:
I think this is solved with 52924a4176f38b8ccac81706c5ecc0241a882858